src/main/java/org/opentripplanner/api/common/RoutingResource.java

```diff
@@ -151,7 +151,33 @@
     @QueryParam("optimize")
     protected OptimizeType optimize;
     
-    /** The set of modes that a user is willing to use, with qualifiers stating whether vehicles should be parked, rented, etc. */
+    /**
+     * <p>The set of modes that a user is willing to use, with qualifiers stating whether vehicles should be parked, rented, etc.</p>
+     * <p>The possible values of the comma-separated list are:</p>
+     *
+     * <ul>
```

Comment:
I'm a little hesitant to create such a list in documentation that we then have to manually maintain. Considering the limited number of "Qualified Modes" actually in use, it seems like we could just define an enum for all these values. But that would be a bigger refactor, I guess this is the best incremental improvement.

Reply:
I spent days debugging an issue with an unhappy customer that traced back to a lack of documentation of what value values were for `leg.mode`. Good, complete docs matter.